automation/src/test/java/org/greenplum/pxf/automation/features/json/JsonTest.java

```diff
@@ -21,15 +22,29 @@
     private final String FILENAME_PRETTY_PRINT = "tweets-pp";
     private final String FILENAME_PRETTY_PRINT_W_DELETE = "tweets-pp-with-delete";
     private final String FILENAME_BROKEN = "tweets-broken";
+    private final String FILENAME_MISMATCHED_TYPES = "supported_primitive_mismatched_types";
 
-    private String[] tweetsFields = new String[] {
+    private String[] tweetsFields = new String[]{
             "created_at text",
             "id bigint",
             "text text",
             "\"user.screen_name\" text",
             "\"entities.hashtags[0]\" text",
             "\"coordinates.coordinates[0]\" float8",
-            "\"coordinates.coordinates[1]\" float8", };
+            "\"coordinates.coordinates[1]\" float8",};
+
+    private String[] supportedPrimitiveFields = new String[]{
+            "type_int int",
+            "type_bigint bigint",
+            "type_smallint smallint",
+            "type_float real",
+            "type_double float8",
+            "type_string1 text",
+            "type_string2 varchar",
+            "type_string3 bpchar",
+            "type_char char",
+            "type_boolean bool",
+            /* "type_bytes bytea",*/};
```

Comment:
let's remove it if we are not going to use it
```suggestion
            };
```